site/docs/v1/tech/core-concepts/_fusionauth_roles.adoc

```diff
@@ -1,7 +1,7 @@
 .FusionAuth application roles
 [cols="1,2,1"]
 |===
-| Name | Id | Description 
```

Comment:
This file should be named `_fusionauth-roles.adoc`, with a `-` not an `_` in the middle.

Reply:
Also, looks like you could in line it, since it doesn't appear to be used anywhere else.